functions/other_isobaric.R

```diff
@@ -0,0 +1,22 @@
+other_isobaric <- function(Combined, metab.list, infile, check.lists) {
+  # not sure what this function does, find metabolites with same mass? Isomeric, not isobaric
```

Comment:
I think anti_join finds differences between two tables -> result = table with different masses?

Reply:
I would try to test this function separately from the other code to find out what it exactly does.